lib/constants.rb

```diff
@@ -23,7 +23,8 @@ module Constants
   1) List contacts
   2) Add contact
   3) Search contact
-  4) Exit the program
+  4) Update contact
+  5) Exit the program
```

Comment:
This commit is an example of shotgun surgery. 

Every time you add a new option to the menu, you have to make little changes to the code in different files.

Also the `EXIT_OPTION` incorrectly replaces the concept of  "items count" sometimes, which is not always interchangeable (for example in the validator).

What happens if the EXIT_OPTION is the first option of the menu? 
How many changes would you have to do?

Can you think of a way to refactor this to avoid shotgun surgery?

Reply:
I've refactored this to separate the validators behaver, so the validator is completely decoupled. 

The controller needs to know the exit option intended, so there for the tests need to know that, as well as the length of the intended actions that relate to the exit option choice.

I have separated the exit choice from the actions count. So now the only change needed to be made next time should just be in the Contants and the bin/app.

Reply:
Nice, looks much better now

(you could still get rid of the count in the Constants, but we can leave it like this for now)